h2/src/main/org/h2/command/Parser.java

```diff
@@ -7400,6 +7403,20 @@ private AlterUser parseAlterUser() {
         throw getSyntaxError();
     }
 
+    private AlterType parseAlterType() {
+        String typeName = readIdentifierWithSchema();
+        if (readIf("ADD")) {
+            Schema schema = getSchema();
+            AlterType command = new AlterType(session, schema);
+            command.setTypeName(typeName);
+            read("VALUE");
```

Comment:
VALUE is a reserved word, it must be read with `read(VALUE)`.

Reply:
Fixed